src/lib_ccx/ccx_encoders_spupng.c

```diff
@@ -650,14 +701,75 @@ int spupng_export_string2png(struct spupng_t *sp, uint32_t *str, FILE* output)
 	
 	FT_GlyphSlot slot = face->glyph;
 
+	// For parsing tags
+	int bracket = 0;
+	int end = 0;
+	int bold = 0;
+	int italics = 0;
+	FT_Face current_face = face;
+
 	// Render characters to image
 	for (uint32_t *iter = str; *iter; ++iter)
 	{
 		uint32_t current_char_code = BigtoLittle32(*iter); // Convert big-endian and little-endian
 
-		if (FT_Load_Char(face, current_char_code, FT_LOAD_RENDER)) continue; // ignore errors
+		// Parse tags like <i> and <b>
+		if (end) {
+			if (current_char_code == '>') {
+				end = 0;
+				bold = 0;
+				italics = 0;
+				bracket = 0;
+				current_face = face;
+			}
+			continue;
+		}
+		if (bracket) {
+			switch (current_char_code)
+			{
+			case 'i':
+				italics = 1;
+
+				// Set to bold and italics
+				if (bold) {
+					current_face = facez;
+				}
+
+				// Set only to italics
+				else {
+					current_face = facei;
+				}
+				break;
+			case 'b':
+				bold = 1;
+
+				// Set to both bold and italics
+				if (italics)
+					current_face = facez;
+				// Set only to bold
+				else
+					current_face = faceb;
+				break;
+			case '/':
+				end = 1;
+				break;
+			case '>':
+				bracket = 0;
+				break;
+			}
+			continue;
+		}
+
+		if (current_char_code == '<') {
+			bracket = 1;
+			continue;
+		}
+
+		slot = current_face->glyph;
```

Comment:
I think it might be better and easier to decide which typeface to use here based on the value of `italics` and `bold`, instead of changing `current_face` in `if`s.